src/transformers/linexp_fit.rs

```diff
@@ -0,0 +1,116 @@
+use crate::transformers::transformer::*;
+
+use conv::prelude::*;
+use macro_const::macro_const;
+
+const INPUT_FEATURE_SIZE: usize = 5;
+
+macro_const! {
+    const DOC: &str = r#"
+Transform LinexpFit features to be more usable
+
+The LinexpFit feature extractor returns the following features:
+- amplitude - kept as is
+- fall_slope - kept as is
```

Comment:
Here, and everywhere in this file, could you please change "fall_slope" to "fall_time"